clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/caching/view/provider/KubernetesV2InstanceProvider.java

```diff
@@ -28,11 +28,14 @@
 import com.netflix.spinnaker.clouddriver.kubernetes.v2.description.manifest.KubernetesManifest;
 import com.netflix.spinnaker.clouddriver.kubernetes.v2.op.job.KubectlJobExecutor;
 import com.netflix.spinnaker.clouddriver.kubernetes.v2.security.KubernetesV2Credentials;
+import com.netflix.spinnaker.clouddriver.model.ContainerLog;
 import com.netflix.spinnaker.clouddriver.model.InstanceProvider;
 import com.netflix.spinnaker.clouddriver.security.AccountCredentialsRepository;
 import com.netflix.spinnaker.clouddriver.security.ProviderVersion;
 import io.kubernetes.client.models.V1Container;
 import io.kubernetes.client.models.V1Pod;
+import java.util.ArrayList;
+import java.util.List;
```

Comment:
Sort `java` imports together, below the rest of the imports.

Reply:
`ctrl-alt-O` does this for you in intellij FYI

Reply:
Thanks, updated! Looks like my Intellij's `ctrl-alt-O` defaults to alphabetizing the imports, but I changed it in settings -> code style to sort by all other imports + newline + java imports.